wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -448,6 +454,17 @@ impl PhysicalDeviceFeatures {
             } else {
                 None
             },
+            shader_int64_atomic: if device_api_version >= vk::API_VERSION_1_1
+                || enabled_extensions.contains(&vk::KhrGetPhysicalDeviceProperties2Fn::name())
+            {
+                Some(
+                    vk::PhysicalDeviceShaderAtomicInt64Features::builder()
+                        .shader_buffer_int64_atomics(true)
```

Comment:
This is checking for VK_KHR_get_physical_device_properties2 not VK_KHR_shader_atomic_int64?